core/src/it/scala/org/ensime/intg/DebugTest.scala

```diff
@@ -274,47 +488,50 @@ trait DebugTestUtils {
     expectMsg(TrueResponse)
 
     val vm = VMStarter(config, className)
```

Comment:
you could pattern match on the left hand side. It is weird that something that looks like a constructor is returning a tuple.